Assets/Prefabs/Interactables/CollectableAmmo.prefab

```diff
@@ -182,8 +183,8 @@ BoxCollider:
   m_ProvidesContacts: 0
   m_Enabled: 1
   serializedVersion: 3
-  m_Size: {x: 0.7, y: 0.95608616, z: 0.7}
-  m_Center: {x: 0, y: 0.7280432, z: 0}
+  m_Size: {x: 2, y: 2, z: 2}
+  m_Center: {x: 0, y: 1, z: 0}
```

Comment:
hva skjedd her? du scala opp collideren? isåfall støtter jeg det veldig :+1:

Reply:
Jepp! Har pgtt så mye feedback om det, så tenkte å gjøre det mens jeg husket, men så glemte jeg å skrive det ned <.<